### ISSUE TITLE
Crash in hv.Labels DynamicMap with hook adding Bokeh Label annotation

### ISSUE THREAD

**Issue by TheoMathurin** [holoviz/holoviews#6178]
#### ALL software version info
Holoviews 1.18.3
Bokeh 3.4.0
Panel 1.4.0

#### Description of expected behavior and the observed behavior
With previous versions of Bokeh (up to 3.3.4 with Panel 1.3.8), I could add a static annotation using a hook on a DynamicMap that itself returned a hv.Labels element. It now triggers a crash and the plot gets updated in an unexpected way.

#### Complete, minimal, self-contained example code that reproduces the issue

```python
import panel as pn
import holoviews as hv
import pandas as pd
from bokeh.models import Label

hv.extension('bokeh')

slider = pn.widgets.IntSlider(name='Slider', start=0, end=1)

df = pd.DataFrame({'text': ['A', 'B'], 'value': [0, 1],
                   'x': [1, 2], 'y': [1, 2]})


def get_plot(value):
    df_ = df[df['value'] == value]
    return hv.Labels(df_, ['x', 'y'], 'text')


def add_annotation(p, element):
    p.state.add_layout(
        Label(x=0, y=0, text='TEXT'))


dmap = hv.DynamicMap(pn.bind(get_plot, slider)).opts(hooks=[add_annotation])

pn.Column(slider, dmap).servable()
```

#### Stack traceback and/or browser JavaScript console output
No error in python, but the following in the browser:
```
Uncaught (in promise) DOMException: FontFaceSet.check: Invalid font shorthand
```

#### Screenshots or screencasts of the bug in action
![labels_bug](https://github.com/holoviz/holoviews/assets/49269671/bed2c6ae-56a4-49ef-8083-c2c8d1dbd938)

**Comment by Hoxbro** [comment]
Likely a regression in Bokeh, cc @mattpap

**Comment by TheoMathurin** [comment]
I'd file an issue there but I'm having a hard time trying to produce a MRE in pure Bokeh.

@mattpap do you have an idea about the root cause of this?

**Comment by mattpap** [comment]
Sorry, I missed this issue. This error most likely originates from bokehjs' font metrics code, but I need to reproduce this and investigate to know more. @TheoMathurin, is this code everything that's involved or maybe there's a theme of some sort involved?

**Comment by TheoMathurin** [comment]
Thanks, yes this example is all that's needed to reproduce.

**Comment by TheoMathurin** [comment]
This appears to be fixed using Bokeh 3.5.2
